src/pyrenew/metaclass.py

```diff
@@ -305,6 +305,11 @@ def __init__(
             If not None, reparameterize sampling
             from the distribution according to the
             given numpyro reparameterizer
+        expand_by_shape : tuple, optional
+            If not None, call :meth:`expand_by()` on the
```

Comment:
```suggestion
            If not None, call :math:`expand_by()` on the
```

Reply:
No, this is `:meth:` for "method"
https://stackoverflow.com/questions/34848253/whats-the-difference-in-behaviour-between-func-and-meth-roles-in-python-sph